apps/arena/native/physics/src/lib.rs

```diff
@@ -232,6 +248,30 @@ fn direction_from_positions(position_a: Position, position_b: Position) -> Direc
     }
 }
 
+#[rustler::nif()]
+fn get_relative_positions_from_entity_with_angles(
```

Comment:
Is this function being used?

Reply:
Nope, this was from a previous implementation for the ultimate, i just removed it